### PR TITLE
fix: [el-cascader] multiple search with v-model can't work

### LINKED ISSUE #3111: [Bug Report] el-cascader multiple search with v-model can't work
### Element Plus version
1.0.2-beta.71

### OS/Browsers version
macbook chrome

### Vue version
3.2.6

### Reproduction Link
https://codepen.io/shangxinbo/pen/YzQXbNy

### Steps to reproduce
Find the official example, and then add the V-model attribute to reproduce it。
in addition.https://github.com/element-plus/element-plus/issues/2296 report before but closed. why? so got the question correct？

### What is Expected?
可选搜索能多选

### What is actually happening?
不能多选

### PR DESCRIPTION
**Pull request by Alanscut**
fix: issue #3111 
same fix with pr #3217 

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by shangxinbo** [comment]
When will it be released？1.1.0-beta10?

**Comment by sxzz** [comment]
> When will it be released？1.1.0-beta10?

We're working on docs website now. We will review and merge it ASAP.

**Comment by shangxinbo** [comment]
> > When will it be released？1.1.0-beta10?
> 
> We're working on docs website now. We will review and merge it ASAP.

so when? The project is going online with this bug.

**Comment by sxzz** [comment]
It works, but I don't understand the code you wrote at all. It is worth noting that `config` is a `ComputedRef`, so `Object.keys(configTemp)` never includes `modelValue`.
WDYT?

**Comment by Alanscut** [comment]
> It works, but I don't understand the code you wrote at all. It is worth noting that `config` is a `ComputedRef`, so `Object.keys(configTemp)` never includes `modelValue`.
> And I changed `config` to `config.value`, but the bug still exists.
> WDYT?

The reason for this issue is that due to the change of `modelvalue`, the tree node is re-rendered and the `uid `changes at the same time. Because the filter list searched by the input text still uses the old node `uid`, all the nodes in filtered list cannot be selected after the first click.
This pr mainly excludes the re-rendering of nodes caused by `modelvalue `changes, so use `configTemp `and `configCheck `without `modelvalue `for comparison.
这个issue产生的原因就是由于modelvalue的改变，导致重新渲染树节点同时uid发生改变，由于输入文字搜索到的过滤列表使用的还是旧的节点uid，所以第一次点击之后无法选中。
这个pr主要就是排除modelvalue改变所导致的节点重新渲染，所以使用除去modelvalue的configTemp和configCheck 进行比较

File: packages/components/cascader-panel/src/index.vue [hunk 1]
```diff
@@ -104,19 +106,32 @@ export default defineComponent({
...
-      manualChecked = false
-      store.value = new Store(options, cfg)
-      menus.value = [store.value.getNodes()]
-
-      if (cfg.lazy && isEmpty(props.options)) {
-        initialLoaded = false
-        lazyLoad(null, () => {
-          initialLoaded = true
+      const configTemp = Object.assign(config)
+      if (Object.keys(configTemp).includes('modelValue')) {
```
Comment (sxzz) [inline packages/components/cascader-panel/src/index.vue:110]: This line always return false. Please check again.

Comment (Alanscut) [reply packages/components/cascader-panel/src/index.vue]: yes，This if is not necessary 。i removed it

**Comment by shangxinbo** [comment]
高低是没有赶上beta10

**Comment by sxzz** [comment]
I used a simpler method, which seems to be able to fix this bug. Can you see if there are any other problems?
(I pushed a commit to your branch. Please remember to `git pull`)

**Review by sxzz — APPROVED** [review]

**Comment by sxzz** [comment]
I modified the name of the variable and added the type.

**Pull request merged**